### PR TITLE
switching css code between different files

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -762,3 +762,354 @@ footer.col-sm-12 {
 .map-sites div h4 {
   padding-left:15px;
 }
+
+    .bs-slider{
```
Comment (MichaelKohler) [inline css/main.css]: can you make sure that this starts at the beginning of the line?
Otherwise looks good, thanks!

**Review by MichaelKohler — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
done

**Comment by ghost** [comment]
reabased to avoid conflicts among branches

**Comment by MichaelKohler** [comment]
There is one conflict as well here. This PR now also includes commits from me, which are already merged. This makes it almost impossible to review your changes now. I'm not sure what went wrong on your side.

To make it easier to review as well as to avoid further conflicts, I'd suggest to apply all your changes from here (but only the one relevant to your feature), to the PR in https://github.com/mozilla/fx-webcompat-campaign/pull/20. Otherwise you will need to rebase every single time after one of the PRs is merged. Let's make it so that https://github.com/mozilla/fx-webcompat-campaign/pull/20 includes all your changes and let's close the other ones.

I'm sorry for this, but otherwise there is a too high risk of destroying existing work.

Thanks for your understanding,
Michael

**Pull request closed without merging**